libs/@guardian/libs/src/consent-management-platform/isConsentOrPay.ts

```diff
@@ -13,7 +14,16 @@ export const getIsConsentOrPay = (): boolean => {
 };
 
 export const isConsentOrPayCountry = (countryCode: CountryCode) => {
-	return consentOrPayCountries.includes(countryCode);
+	return countryCode in consentOrPayCountries;
+};
+
+export const getConsentOrPayCurrency = (countryCode: CountryCode): Currency => {
+	if (!isConsentOrPayCountry(countryCode)) {
```

Comment:
It could be easier to do below as I'm not sure if the type assertion is necessary. 

return consentOrPayCountries[countryCode] ?? 'GBP';

Reply:
I have simplified it, but couldn't get quite as far as your suggestion as it was giving an ESLint error